File: Python/771.py
# 771. Jewels and Stones
# https://leetcode.com/problems/jewels-and-stones/

class Solution:
  def numJewelsInStones(self, jewels: str, stones: str) -> int:
    out = 0

    for s in stones:
      if (jewels.find(s) > -1):
        out += 1

    return out

File: Rust/771.rs
/**
 * 771. Jewels and Stones
 * https://leetcode.com/problems/jewels-and-stones/
 */

impl Solution {
  pub fn num_jewels_in_stones(jewels: String, stones: String) -> i32 {
    let mut out = 0;

    for s in stones.chars() {
      if jewels.find(s) != None {
        out += 1;
      }
    }

    return out;
  }
}
